scenarios/19_Sky_Kingdom.cfg

```diff
@@ -41,7 +41,7 @@
 
         type=EoMa_Master_of_Water
         id=MasterWater
-        name= _ "Master of Water"
+        name= _ "Aqauticus"
```

Comment:
Perhaps this was meant to be `Aquaticus`?

Reply:
Indeed, it is a typo. Thanks for spotting it :) Will be fixed next time.

Reply:
fixed in my PR